packages/node_modules/@webex/internal-plugin-wdm/src/device/service-model.js

```diff
@@ -2,9 +2,10 @@
  * Copyright (c) 2015-2019 Cisco Systems, Inc. See LICENSE file.
  */
 
+import url from 'url';
```

Comment:
Generally we want to keep these in alphabetical order.

Reply:
I'll update the PR based on the comments above. Thanks @mattnorris